src/com/owncloud/android/ui/fragment/ShareFileFragment.java

```diff
@@ -410,6 +410,15 @@ private boolean isPublicShareDisabled() {
      */
     private void updateListOfPublicLinks() {
 
+        // Hide warning about public links if not enabled in server
```

Comment:
The comment is for the next "paragraph" of code, no just the variable. The code executed conditionally is on ```(!showDisplayWarning)```, negated. And what it does is setting the visibility of the message to ```GONE```, so it's effectively hiding it (or doing nothing).